components/apimgt-extensions/org.wso2.carbon.apimgt.application.extension.api/src/main/java/org/wso2/carbon/apimgt/application/extension/api/filter/Permission.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
this code block was refactored and bought to this component. fixed with https://github.com/wso2/carbon-device-mgt/pull/403/commits/3140c36f7b80a1bfe81be075f84809e9b8cddc3e